### PR TITLE
add currency input

### PR DESCRIPTION
**Pull request by Samsinite**

### CODE REVIEW

File: app/initializers/text-mask-addons.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import createNumberMask from 'npm:text-mask-addons/dist/createNumberMask';
+
+export default {
+  name: 'ember-dnd-core',
+  initialize() {
+    console.assert(createNumberMask);
```
Comment (chadzink) [inline app/initializers/text-mask-addons.js]: Never seen this before 😄. Looks like it works in all the browsers too.

**Comment by jweakley** [comment]
👎  No documentation added to the README.md. If you don't get to it @Samsinite, I'll see if I can add it at some point today.

**Comment by jweakley** [comment]
I added the update to the README. @Samsinite, could you give that a once over to make sure I got the documentation correct. 👍  once that is done.

**Comment by Samsinite** [comment]
Thanks @jweakley. I added a few more comments, can you glance at them for me and let me know if you have any feedback?

**Comment by Samsinite** [comment]
Err added a few more notes to the README

**Comment by jweakley** [comment]
Only thing would be moving the use example to the top of the section to match the other components.

**Comment by Samsinite** [comment]
I actually like specifying the requirements before showing usage. Seems more clear to me as a developer because that is how I'm used to reading documentation.

**Comment by jweakley** [comment]
I have no strong preference either way as long as it is consistent. If you would like to have requirements before usage, then we should update all the others to be consistent.

**Comment by Samsinite** [comment]
Me neither, I'll just make this one match the others. Took a look at how ember documents, and they do it like you did, so let's just follow that convention for ember addons.

**Comment by jweakley** [comment]
I think the reasoning for usage first is that most developers are looking for a copy/paste thing and tweak to suit their specific use case.

**Comment by Samsinite** [comment]
And that is fair reasoning.

**Pull request merged**